### DISCUSSION TITLE
BUILD FAILED masOC 13.0.1 using ruby-build 20230309-1-gf3c1ccc

### DISCUSSION THREAD

**Discussion by thespica93** [rbenv/ruby-build#2166] [Build failures]
Sorry I'm new to this. I can get it to install properly 
![Screenshot 2023-03-14 at 9 57 08 PM](https://user-images.githubusercontent.com/42631192/225185155-46ee9c39-f062-4df4-a773-64ad3f1a6aa3.png)

**Comment by mislav** [comment]
1. The output says that the version 3.2.1 already exists on your system. Are you sure you need to reinstall it from scratch?
2. There is not enough information here to debug the failure. The output says `Results logged to <file>` which points to the full build log. If you open the file (for instance with `less <file>`), you can inspect the last portion of it and see if there is any informative error in there.
3. In general, when pasting output to issue reports, it's better to copy-paste the text itself (and enclose it in Markdown code blocks) than to make a screenshot. Unlike screenshots, text is be searchable.

Good luck!
